src/analysis/useless_initializer.d

```diff
@@ -25,45 +32,128 @@ final class UselessInitializerChecker : BaseAnalyzer
 private:
 
     enum key = "dscanner.useless-initializer";
+
     version(unittest)
+    {
         enum msg = "X";
+    }
     else
+    {
         enum msg = `Variable %s initializer is useless because it does not differ from the default value`;
+    }
 
     static immutable strDefs = [`""`, `""c`, `""w`, `""d`, "``", "``c", "``w", "``d", "q{}"];
     static immutable intDefs = ["0", "0L", "0UL", "0uL", "0U", "0x0", "0b0"];
 
+    HashMap!(string, bool) _structCanBeInit;
+    DynamicArray!(string) _structStack;
+    DynamicArray!(bool) _inStruct;
+    DynamicArray!(bool) _atDisabled;
+    DynamicArray!(bool) _inTest;
```

Comment:
Hmm I am not a huge fan of using `_` if there's a proper type system with `private` in place.